api/v1alpha1/seedgenerator_types.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Apparently the new code generator assumes all types defined in a file named `types.go`.  We had to merge talm ones in this PR: https://github.com/openshift-kni/cluster-group-upgrades-operator/pull/709

Reply:
https://github.com/kubernetes/code-generator/blob/7855250cd36120e10dc5bbf693ac4bdf42b7541f/kube_codegen.sh#L399

https://github.com/kubernetes/code-generator/blob/7855250cd36120e10dc5bbf693ac4bdf42b7541f/kube_codegen.sh#L555

Reply:
Interesting... thanks

Reply:
Created a separate dir